lib/pure/random.nim

```diff
@@ -669,6 +681,20 @@ when isMainModule:
     doAssert a[0] == 1
     doAssert a[1] == 0
 
+    const b = @[0, 1]
+    let c = shuffled(b)
+    doAssert b[0] == 0
+    doAssert b[1] == 1
+    doAssert c[0] == 1
+    doAssert c[1] == 0
+
+    const d = [0, 1]
+    let e = shuffled(b)
```

Comment:
you shuffel twice `b` here. `d` is unused.

Reply:
@krux02 thanks, I fixed.